packages/probing-frontend/src/pages/MonitoringDetailPage/MonitoringDetail.page.tsx

```diff
@@ -0,0 +1,76 @@
+import { MonitoringQueries } from '@/api/monitoring/monitoring.hooks'
+import { Link, useParams } from '@/router'
+import { MonitoringEserviceTelemetry } from './components/MonitoringEserviceTelemetry'
+import { MonitoringEserviceProbing } from './components/MonitoringEserviceProbing'
+import { MonitoringEserviceDetail } from './components/MonitoringEserviceDetail'
+import { Box } from '@mui/system'
+import { useTranslation } from 'react-i18next'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { Typography } from '@mui/material'
+import { useLoadingOverlay } from '@/stores'
+
+export const MonitoringDetailPage = () => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const { showOverlay, hideOverlay } = useLoadingOverlay()
+  const { id: eserviceId } = useParams<'MONITORING_DETAIL'>()
+  const { data: eservicesDetail } = MonitoringQueries.useGetEserviceData(
+    {
+      eserviceId,
+    },
+    { suspense: false }
```

Comment:
Since we are not using suspense at all, I wonder if making it false by default would be better for our use case. What do you think?